lib/internal/url.js

```diff
@@ -545,7 +546,6 @@ class URL {
     // toUSVString is not needed.
     input = `${input}`;
     this[context] = new URLContext();
-    this.#onParseComplete = FunctionPrototypeBind(this.#onParseComplete, this);
 
```

Comment:
Let's move this to its own PR: https://github.com/nodejs/node/pull/46870
```suggestion
this.#onParseComplete = FunctionPrototypeBind(this.#onParseComplete, this);

```